xml/System/Uri.xml

```diff
@@ -188,6 +188,36 @@ Uri uri1 = new Uri("C:/test/path/file.txt") // Implicit file path.
 Uri uri2 = new Uri("file:///C:/test/path/file.txt") // Explicit file path.
 ```
  These implicit file paths are not compliant with the URI specification and so should be avoided when possible. When using .NET Core on Unix-based systems, implicit file paths can be especially problematic, because an absolute implicit file path is *indistinguishable* from a relative path. When such ambiguity is present, <xref:System.Uri> default to interpreting the path as an absolute URI.
+
+## Security Considerations
+
+Because of security concerns, your application should use caution when accepting <xref:System.Uri> instances from untrusted sources and with `dontEscape` set to `true`. You can check a URI string for validity by calling the <xref:System.Uri.IsWellFormedOriginalString%2A> method.
```

Comment:
The preview looks good now. One last thing I noticed - it isn't clear to me where the `dontEscape` parameter comes into play just from reading this. I now see it is used in two constructor overloads, but they are both marked obsolete, and that parameter is always `false`. But if you can set it to true, perhaps in an older version?, I think it should say:

```suggestion
Because of security concerns, your application should use caution when accepting <xref:System.Uri> instances from untrusted sources and with `dontEscape` set to `true` in the [constructor](xref:System.Uri.%23ctor(System.String,System.Boolean)). You can check a URI string for validity by calling the <xref:System.Uri.IsWellFormedOriginalString%2A> method.
```

Otherwise should it just say:

```suggestion
Because of security concerns, your application should use caution when accepting <xref:System.Uri> instances from untrusted sources. You can check a URI string for validity by calling the <xref:System.Uri.IsWellFormedOriginalString%2A> method.
```

Reply:
I copy pasted this paragraph from the removed section. The changes look reasonable to me.